Dockerfile

```diff
@@ -17,8 +17,42 @@ RUN go build -ldflags "-X main.version=${VERSION}" -o auroraboot
 FROM fedora:$FEDORA_VERSION AS default
 RUN dnf -y update
 ## ISO+ Arm image + Netboot + cloud images Build depedencies
-RUN dnf in -y bc jq genisoimage docker sudo parted e2fsprogs erofs-utils binutils curl util-linux udev rsync \
-    dosfstools mtools xorriso lvm2 zstd sbsigntools squashfs-tools kpartx grub2 openssl
+RUN dnf in -y bc \
+              binutils \
+              curl \
+              docker \
+              dosfstools \
+              e2fsprogs \
+              erofs-utils \
+              gdisk \
+              genisoimage \
+              git \
+              grub2 \
+              jq \
+              kpartx \
+              lvm2 \
+              mtools \
+              openssl \
+              parted \
+              qemu-img \
+              qemu-system-x86 \
+              qemu-tools \
+              rsync \
+              sbsigntools \
+              squashfs-tools \
+              sudo \
+              udev \
+              util-linux \
+              xfsprogs \
+              xorriso \
+              zstd \
+              python3-cryptography python3-pefile # ukify deps
```

Comment:
this were dropped already, we dont use ukify anymore. Is this list updated?

Also dont install docker here if we are using the upstream one below :D